dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -147,6 +147,12 @@ func addAdditionalFiles(srcPath, dstPath, version string) {
 		"integration/kubernetes/conf/alluxio.properties.template",
 		// FUSE
 		"integration/fuse/bin/alluxio-fuse",
+		"integration/mesos/bin/alluxio-env-mesos.sh",
```

Comment:
Could you comment `// MESOS` above this?

Reply:
done, thanks